README.adoc

```diff
@@ -236,31 +236,28 @@ PersonServiceIT.java
+The following tests are added: [hotspot=testMinSizeName file=1]`testMinSizeName()`, [hotspot=testMinAge file=1]`testMinAge()`, [hotspot=testGetPerson file=1]`testGetPerson()`, [hotspot=testGetAllPeople file=1]`testGetAllPeople()`, and [hotspot=testUpdateAge file=1]`testUpdateAge()`.
 
 
 Save the changes, and  press the `enter/return` key in your console window to run the tests.
 
 == Testing outside of development mode
 
-Running the tests against a pre-started Liberty server is very convenient for local development, but this may be tedious to set up in non-dev scenarios such as a CI/CD pipeline. For this reason, MicroShed Testing can start and stop the application runtime before and after the tests are run. This is primarily accomplished using Docker and Testcontainers.
+Running tests in development mode is convenient for local development, but it can be tedious to test against a running Open Liberty server in non-development scenarios such as CI/CD pipelines. For this reason, MicroShed Testing can start and stop the application runtime before and after the tests are run. This process is primarily accomplished by using Docker and Testcontainers.
 
-To test outside of development mode, first exit development mode by typing `q` in the shell session where you ran the server and then press the `enter/return` key.
+To test outside of development mode, exit development mode by typing `q` in the shell session where you ran the server, and then press the `enter/return` key.
 
-Next, run the tests from a "cold start" by running the command:
+Next, use the following Maven goal to run the tests from a cold start:
 [role='command']
 ```
 mvn verify
 ```
 
-Running tests from a cold start will take a little longer than running the tests from development mode, since the application runtime needs to be started each time. However, the benefits of this mode are:
-
-- A clean instance is used on each test run, for consistent results
-- Automatically hooks into many existing build pipelines that are set up to run the `integration-test` phase
+Running tests from a cold start takes a little longer than running tests from development mode because the application runtime needs to start each time. However, tests that are run from a cold start use a clean instance on each run to ensure consistent results. These tests also automatically hook into existing build pipelines that are set up to run the `integration-test` phase.
 
 == Sharing configuration across multiple classes
 
-Typically, projects have multiple test classes that all exercise the same type of application deployment. For these cases, it is useful to reuse existing configuration and application lifecycle across multiple test classes.
+Typically, projects have multiple test classes that all use the same type of application deployment. For these cases, it is useful to reuse existing configuration and application lifecycles across multiple test classes.
```

Comment:
here `lifecycle` should be singular because we're talking about reusing a single application lifecycle

Reply:
So, `For these cases, it is useful to reuse an existing configuration and application lifecycle across multiple test classes.`?

Reply:
@aguibert is it ^ OK?

Reply:
yep looks good